waspc/src/Wasp/AppSpec/Page.hs

```diff
@@ -8,7 +8,7 @@ where
 import Data.Data (Data)
 import Wasp.AppSpec.Core.Decl (IsDecl)
 
-data Page = Page
+newtype Page = Page
```

Comment:
Same as above, I know for fact this one will be expanded soon, so I didn't want to make it newtype since it is really not, it is data that for now has just one field. I would turn off that linter tool and assume we know when to use data and when to use newtype.